utils/libnetwork/Makefile

```diff
@@ -13,8 +14,7 @@ GO_PKG_BUILD_PKG:= \
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://$(GO_PKG)
-PKG_SOURCE_VERSION:=e7933d41e7b206756115aa9df5e0599fc5169742
-PKG_MIRROR_HASH:=48638648bfd2b249f8e9cc32b5ec295a64e61fcb7cf635ca1a88809662167374
+PKG_MIRROR_HASH:=f5be4cb87493393a7cd379d8a3e57690523f6e0c500483c660fd4f6a6f93772e
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.xz
```

Comment:
Completely unneeded. download.mk takes care of this.

Reply:
Removed